site/.eslintrc.yaml

```diff
@@ -114,6 +114,9 @@ rules:
   react/jsx-curly-brace-presence:
     - error
     - children: ignore
+  # https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#eslint
+  react/jsx-uses-react: "off"
+  react/react-in-jsx-scope: "off"
```

Comment:
Turning off these ESLint rules ([as suggested by React](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#eslint)) so we don't hear it complain.